hack/components/bump-ovs-cni.sh

```diff
@@ -107,13 +108,15 @@ OVS_TAG=$(git-utils::get_component_tag ${OVS_PATH})
 echo 'Get ovs-cni-plugin image name and update it under CNAO'
 OVS_PLUGIN_IMAGE=quay.io/kubevirt/ovs-cni-plugin
 OVS_PLUGIN_IMAGE_TAGGED=${OVS_PLUGIN_IMAGE}:${OVS_TAG}
-sed -i "s#\"${OVS_PLUGIN_IMAGE}:.*\"#\"${OVS_PLUGIN_IMAGE_TAGGED}\"#" \
+OVS_PLUGIN_IMAGE_DIGEST="$(docker-utils::get_image_digest "${OVS_PLUGIN_IMAGE_TAGGED}" "${OVS_PLUGIN_IMAGE}")"
+sed -i -r "s#\"${OVS_PLUGIN_IMAGE}(@sha256)?:.*\"#\"${OVS_PLUGIN_IMAGE_DIGEST}\"#" \
     pkg/components/components.go \
     test/releases/${CNAO_VERSION}.go
 
-echo 'Get ovs-cni-marker image name and update it under CNAO'
+echo -r 'Get ovs-cni-marker image name and update it under CNAO'
```

Comment:
AFAIK [echo](https://linux.die.net/man/1/echo) does not have `-r` flag. please remove